pkg/ovs/openflow/utils.go

```diff
@@ -727,7 +744,7 @@ func nxActionDecTTLToString(action openflow15.Action) string {
 
 func nxActionConjunctionToString(action openflow15.Action) string {
 	a := action.(*openflow15.NXActionConjunction)
-	actionStr := fmt.Sprintf("conjunction:(%d,%d/%d)", a.ID, a.Clause, a.NClause)
+	actionStr := fmt.Sprintf("conjunction(%d,%d/%d)", a.ID, a.Clause+1, a.NClause)
```

Comment:
I don't get this , why `+1` is needed?

Reply:
This is because the value of `Clause` field used in OpenFlow message starts from 0, but the output used in ovs-ofctl dump-flows starts from 1. We expect message string to be consistent with ovs command line.

Reply:
yes.